tools/lkl/tests/net.sh

```diff
@@ -81,7 +83,8 @@ if ! [ -z $DST ]; then
     ${SUDO} ip link set dev ${IFNAME} promisc off
 
     echo "== macvtap (LKL net) tests =="
-    ${SUDO} ip link add link ${IFNAME} name lkl_vtap0 type macvtap mode passthru
+    ${SUDO} ip link add link ${IFNAME} name lkl_vtap0 \
```

Comment:
Should we ignore it only for Android?

Reply:
there is one more check with /dev/tap* right after this line, so it should be okay I think.